src/main/java/rs/teslaris/core/model/document/PublicationSeries.java

```diff
@@ -0,0 +1,55 @@
+package rs.teslaris.core.model.document;
+
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.Where;
+import rs.teslaris.core.model.commontypes.BaseEntity;
+import rs.teslaris.core.model.commontypes.LanguageTag;
+import rs.teslaris.core.model.commontypes.MultiLingualContent;
+
+
+@Getter
+@Setter
+@Entity
+@Table(name = "publication_series")
+@Where(clause = "deleted=false")
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
+public abstract class PublicationSeries extends BaseEntity {
+
+    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    private Set<MultiLingualContent> title;
+
+    @Column(name = "e_issn", unique = true)
+    private String eISSN;
+
+    @Column(name = "print_issn", unique = true)
+    private String printISSN;
+
+    @OneToMany(mappedBy = "publicationSeries", fetch = FetchType.LAZY, orphanRemoval = true)
+    private Set<PersonJournalContribution> contributions;
```

Comment:
Set<PersonPublicationSeriesContribution>

Reply:
I'm not sure what you meant by this :smile:

Reply:
It should be Set<PersonPublicationSeriesContribution>, right?

Reply:
@ivanmrsulja apologies, I didn't check the preview of my comments. Please check my above comments again :-)

Reply:
Yes, it should be `Set<PersonPublicationSeriesContribution>` :smile: i changed it